index.bs

```diff
@@ -9911,7 +9911,37 @@ validity of a given {{AudioWorkletProcessor}} subclass.
 			<em>Return type:</em> {{boolean}}
 		</div>
 </dl>
-	
+
+<pre class="example" highlight="js" title="Subclassing AudioWorkletProcessor">
```

Comment:
Bikeshed lets you use `<xmp>` instead of `<pre>` so you can use `<` directly instead of using the html entity.

Reply:
Done.